gap/examples.gi

```diff
@@ -738,3 +736,33 @@ function(filt, m)
   SetIsBipartiteDigraph(D, true);
   return D;
 end);
+
+InstallMethod(StackedBookDigraphCons,
+"for IsMutableDigraph and two positive integers",
+[IsMutableDigraph, IsPosInt, IsPosInt],
+function(filt, m, n)
+  local book;
+  book := ChainDigraph(IsMutable, n);
+  book := DigraphSymmetricClosure(book);
```

Comment:
```suggestion
  book := DigraphSymmetricClosure(ChainDigraph(IsMutable, n));
```

I thought that maybe a single line would be neater. It works on my computer like that :)

Reply:
Great idea. I will be changing that now! Thank you for the review :D